src/app.rs

```diff
@@ -256,7 +256,35 @@ impl<T: Layout> App<T> {
                             window_id,
                             &mut frame_event_info,
                             &ui_state_cache,
-                            &mut self.app_state);
+                            &mut self.app_state
+                        );
+                    }
+                }
+
+                // Scroll for the scrolled amount for each node that registered a scroll state.
+                if let MouseState { cursor_pos: Some(pos), scroll_x, scroll_y, .. } = window.state.mouse_state {
+                    if scroll_x + scroll_y != 0.0 {

```

Comment:
1. Agreed, will do as requested.
2. I am aware of the issues. But here it will always be 0.0 except if there occured some scroll delta, x or y wise. So comparing with 0.0 is 100% fine. Also comparing with a threshold will not scroll for really small deltas which is not preferable imo. So I have to disagree with this.
3. Ok, I agree with the case where both value cancel out each other. But summing the f32::abs() will do.
4. How do you mean invert? I gotta unpack the values somehow before I can compare/use them.